index.html

```diff
@@ -69,16 +68,28 @@
     <div id="bar-chart-large"></div>
 
     <script type="text/javascript">
-      $.get( "/data", function(data) {
-        getBarChart({
-          data: data,
-          width: 800,
-          height: 600,
-          xAxisLabel: 'System',
-          yAxisLabel: 'Valid',
-          containerId: 'bar-chart-large'
-        })
-      });
+      $.urlParam = function(name){
```

Comment:
When you find code, it's a good practice to add a comment with a link about where it came from. That can give context for developers who look at the code later.